package.json

```diff
@@ -22,8 +22,9 @@
     "build:dev": "webpack --mode development",
     "build:watch": "npm run build:dev -- --watch",
     "clean": "del-cli _site tests/_artifacts",
-    "format": "prettier --write . && eslint . --fix",
-    "lint": "prettier --check . && eslint .",
+    "prettier": "prettier --ignore-unknown \"**/*.!(js|jsx|mjs|cjs|ts|tsx|mts|cts)\"",
+    "format": "npm run prettier -- --write . && xo --fix",
+    "lint": "npm run prettier -- --check . && xo",
```

Comment:
Would you mind to separate `prettier` and `xo` commands like in https://github.com/simple-icons/simple-icons/pull/10739?

Reply:
What do you mean? I already moved the `prettier --ignore-unknown \"**/*.!(js|jsx|mjs|cjs|ts|tsx|mts|cts)\"` to the `prettier` script.

Reply:
See the second statement of the linked PR:

> Shows that `xo --fix` is being executed in format script. Currently, after Prettier is executed `xo --fix` runs but no output is shown, which seems that Prettier is stopped at some point.

Currently, after running `npm run prettier` the output looks freeze until XO ends, which looks like Prettier is stopped at some point.